examples/chain_demo.py

```
@@ -0,0 +1,180 @@
+
+async def demo_context_sharing():
+    """演示上下文共享"""
+    print("🎯 上下文共享示例")
+
+    # 创建共享上下文
+    context = ChainContext({"project": "nanoagent", "goal": "代码分析"})
+
+    # 创建提示链
+    chain = PromptChain(
+        [
+            ChainStep("步骤1", "分析项目 {project}，目标是 {goal}"),
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)